tests/test_unit_propagation.py

```
@@ -0,0 +1,221 @@
+    solve, and returns the path for verification.
+    """
+    import shutil
+
+    from temoa._internal.temoa_sequencer import TemoaSequencer
+    from temoa.core.config import TemoaConfig
+
+    # Skip if source database or solver doesn't exist
+    if not VALID_UNITS_DB.exists():
+        pytest.skip('Valid units database not available')
+
+    is_available, location = TemoaConfig._check_solver_availability('appsi_highs')
```

Ruff (0.14.8)

Unpacked variable `location` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)